src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource_Async.cs

```diff
@@ -93,7 +93,7 @@ private partial class SuggestedActionsSource : IAsyncSuggestedActionsSource
                     // especially important as we are sending disparate requests for diagnostics, and we do not want the
                     // individual diagnostic requests to redo all the work to run source generators, create skeletons,
                     // etc.
-                    using var _1 = RemoteKeepAliveSession.Create(document.Project.Solution, _listener);
+                    using var _1 = await RemoteKeepAliveSession.CreateAsync(document.Project.Solution, cancellationToken).ConfigureAwait(false);
```

Comment:
the async entrypoint is nicer for async clients, ensuring that we don't proceed until actually makign the remote host client, and issuing the keepalive call.